src/runtime/platform.cpp

```diff
@@ -40,7 +40,7 @@ extern "C" LEAN_EXPORT uint8 lean_system_platform_emscripten(obj_arg) {
 #endif
 }
 
-extern "C" object * lean_get_githash(obj_arg) { return lean_mk_string(LEAN_GITHASH); }
+extern "C" object * lean_get_githash(obj_arg) { return lean_mk_ascii_string(LEAN_GITHASH); }
```

Comment:
Also unlikely, but since the source is out of our (local) control, better safe than sorry.
```suggestion
extern "C" object * lean_get_githash(obj_arg) { return lean_mk_utf8_string(LEAN_GITHASH); }
```